### ISSUE TITLE
Add eval stats to Visdom graph

### ISSUE THREAD

**Issue by rwightman** [ikostrikov/pytorch-a2c-ppo-acktr-gail#121]
The evaluation that was added is very helpful. I was looking at adding, if present, the eval reward to the Visdom graph.

However, with the way that the baselines Monitor is written, opening and closing the envs with each eval run overwrites the monitor csv files. Without changing the monitor code, the easiest way to allow this to work would be to create the eval envs once, reset and reuse them for each eval. Then the monitor would append to the same logs and the addition of the data to the visdom plot would be straightforward.

Would you be interested in this change or do you have other plans in mind?

**Comment by ikostrikov** [comment]
I was going to get rid of plotting from csv files from Monitor since I found a way to get rewards from infos.

**Comment by ikostrikov** [comment]
Visdom is not deprecated.

Results can be visualized in ipynb.

I'm planning to add tensorflowx support soon.
